test/sr_test/sr_sessions.erl

```diff
@@ -9,7 +9,7 @@
 -type user() :: binary().
 -type agent() :: binary().
 
--opaque session() ::
+-type session() ::
```

Comment:
Please add a comment stating the reason for this change (maybe with a link to the bug report, if possible)